content/applications/marketing/email_marketing.rst

```diff
@@ -1,440 +1,792 @@
+- :guilabel:`In Queue`: the email is scheduled to be sent at a later date.
+- :guilabel:`Sending`: the email is currently being sent to its recipients.
+- :guilabel:`Sent`: the email has already been sent to its recipients.
 
-In each stage, there are drag-and-drop cards representing the email campaigns that have been
-created, and the stage they are in represents the current status of that mailing.
+In each stage, there are drag-and-drop cards representing the emails that have been created/sent,
+and the stage they are in represents the current status of that mailing.
 
-Each mailing block on the :guilabel:`Email Marketing` dashboard provides key information related to
-that specific email.
+Each card on the :guilabel:`Mailings` dashboard provides key information related to that specific
+email.
 
-When the cursor hovers over the upper-right corner of an email campaign card, three vertical dots
-(:guilabel:`⋮`) appear. When clicked, a mini drop-down menu reveals the option to color-code the
-email :guilabel:`Delete` the email altogether, or :guilabel:`Archive` the message for potential
-future use.
+When the cursor hovers over the upper-right corner of an email campaign card, :guilabel:`⋮ (three
+vertical dots)` appear. When clicked, a mini drop-down menu reveals the option to color-code the
+email, :guilabel:`Delete` the email, or :guilabel:`Archive` the message for potential future use.
 
 .. image:: email_marketing/three-dot-dropdown.png
    :align: center
    :alt: View of the three-dot drop-down menu on the Odoo Email Marketing dashboard.
 
-In the upper-right corner of the main :guilabel:`Email Marketing` dashboard, there are other view
-options to choose from: :guilabel:`List` and :guilabel:`Graph`.
+.. _email_marketing/calendar-view:
 
-List view
----------
+Calendar view
+-------------
 
-The :guilabel:`List` view (represented by the :guilabel:`☰ (three vertical lines)` icon) provides
-the same key information, but in a classic list format.
+The calendar view, represented by a :guilabel:`📆 (calendar)` icon, can be accessed in the
+upper-right corner of the :guilabel:`Mailings` dashboard in the :guilabel:`Email Marketing` app.
 
-.. image:: email_marketing/list-view.png
+While in calendar view, a monthly calendar (by default), shows when the mailings have been sent or
+are scheduled to be sent.
+
+.. image:: email_marketing/calendar-view.png
    :align: center
-   :alt: How the List view appears in the Odoo Email Marketing application.
+   :alt: Calendar view of the mailings dashboard in the Email Marketing application.
+
+The current date is represented by a :guilabel:`🔴 (red circle)` icon over the date on the calendar.
+
+To the right of the calendar, the options to filter the results by :guilabel:`Responsible` and/or
+:guilabel:`Status` are available, via checkboxes.
+
+.. tip::
+   To hide the right sidebar, click the :guilabel:`(white and black box)` icon, located above the
```

Comment:
This Odoo UI icon is called `panel-right`
![Screenshot 2024-02-23 at 12 03 01 PM](https://github.com/odoo/documentation/assets/5496809/b7ff7da3-cbcf-4b91-af67-de0ad6749383)

```suggestion
   To hide the right sidebar, click the :guilabel:`(panel-right)` icon, located above the
```